inc/settings.php

```diff
@@ -147,6 +147,7 @@ public function get_default_option_values() {
 			'fc_compat_theme_betheme_output_checkout_steps_section'         => 'no',
 			'fc_compat_theme_thegem_output_checkout_steps_section'          => 'no',
 			'fc_compat_theme_go_enable_account_wide_layout'                 => 'yes',
+			'fc_compat_theme_blocksy_force_color_mode_switch'               => 'no',
```

Comment:
Remove this option from the plugin integration settings, since we should always follow the color mode set by the theme.